modules/oc-mirror-installing-plugin.adoc

```diff
@@ -12,6 +12,8 @@ To use the oc-mirror OpenShift CLI plugin to mirror registry images, you must in
 .Prerequisites
 
 * You have installed the OpenShift CLI (`oc`).
+* Make sure that the `umask` is set to `0022` on the operating system that uses oc-mirror.
+* Make sure you are using the correct `RHEL8` binary if your operating system is `RHEL8`, and `RHEL9` for `RHEL9` if your operating system is `RHEL9`. As `RHEL9` is FIPS compliant and has different linking for static and dynamic libraries.
```

Comment:
@subhtk should we change like `Make sure you are using the correct `RHEL8` binary if your operating system is `RHEL8` and `RHEL9` binary if your operating system is `RHEL9` ? Not sure if `and `RHEL9` for `RHEL9`` is okay.